### PR TITLE
Add necessary fields to `SendEmailRequest`

### LINKED ISSUE #56: Stabilize `send-email` API contract
I'm thinking the endpoint could look something like the following. 

```typescript
POST /api/send-email

{
  "relatedIssueId": int, // the issue a changemaker is writing their legislators about
  "emailBody": string,
  "contactedBioguideIds": string[] // the list of legislators the email is being sent to
}
```

### PR DESCRIPTION
**Pull request by kevincianfarini**
This commit adds fields necessary to make a (S)CWC contact http call.
Some of this information has already been collected from the user as
part of finding their members of congress -- we should reuse that
information versus redundantly asking for it again.

The `relatedTopics` and `contactedBioguideIds` always expect to have at
least one element in them.

Closes #56

### CODE REVIEW

File: backend/src/main/kotlin/org/climatechangemakers/act/feature/action/model/SendEmailRequest.kt [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+  val postalCode: String,
+  val relatedTopics: List<Topic>,
+  val emailSubject: String,
   val emailBody: String,
+  val relatedIssueId: Long,
   val contactedBioguideIds: List<String>,
-)
\ No newline at end of file
+) {
+  init {
```
Comment (EvanHahn) [inline backend/src/main/kotlin/org/climatechangemakers/act/feature/action/model/SendEmailRequest.kt:23]: Do we want to add additional validations? For example, `firstName` probably has to be nonempty.

Also, is there any security risk with any of these? I could imagine someone knocking the server over with a giant array or a long name.

File: backend/src/main/kotlin/org/climatechangemakers/act/feature/action/model/SendEmailRequest.kt [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
   val originatingEmailAddress: String,
-  val relatedIssueId: Long,
+  val title: Prefix,
+  val firstName: String,
+  val lastName: String,
+  val streetAddress: String,
+  val city: String,
+  val state: RepresentedArea,
+  val postalCode: String,
+  val relatedTopics: List<Topic>,
```
Comment (EvanHahn) [inline backend/src/main/kotlin/org/climatechangemakers/act/feature/action/model/SendEmailRequest.kt:17]: Is this resilient given that representatives might change the topic strings on a whim? (Maybe this comment is out of scope for this PR.)

**Review by EvanHahn — APPROVED** [review]
LGTM. Left some comments that shouldn't block a merge.

File: backend/src/main/kotlin/org/climatechangemakers/act/feature/action/model/SendEmailRequest.kt (hunk 2 above)
Comment (kevincianfarini) [reply backend/src/main/kotlin/org/climatechangemakers/act/feature/action/model/SendEmailRequest.kt:17]: These topics are sanctioned by the CWC api!

File: backend/src/main/kotlin/org/climatechangemakers/act/feature/action/model/SendEmailRequest.kt (hunk 1 above)
Comment (kevincianfarini) [reply backend/src/main/kotlin/org/climatechangemakers/act/feature/action/model/SendEmailRequest.kt:23]: Great point!
